jetson/nav/search/searchStateMachine.cpp

```diff
@@ -10,46 +10,49 @@
 #include <time.h>
 #include <cmath>
 
-// Constructs an SearchStateMachine object with roverStateMachine
-SearchStateMachine::SearchStateMachine(StateMachine* roverStateMachine )
-    : roverStateMachine( roverStateMachine ) {}
+// Constructs an SearchStateMachine object with roverStateMachine, mRoverConfig, and mPhoebe
+SearchStateMachine::SearchStateMachine(StateMachine* roverStateMachine, Rover* rover, const rapidjson::Document& roverConfig)
+    : roverStateMachine( roverStateMachine ) 
+    , mPhoebe( rover ) 
+    , mRoverConfig( roverConfig ) {}
+
```

Comment:
When I compile I get a warning here. Switching the order of the initializer list so it matches the order of initialization in the hpp file (switch mPhoebe and mRoverConfig) should make the warning go away. I know it doesn't make a difference but its probably just a best practice to not have warnings